src/components/ResultItem/_result-item.scss

```diff
@@ -1,13 +1,13 @@
 .result-item {
   border-top: 1px solid var(--colour-concrete-10);
-  padding-bottom: rem(25);
-  padding-top: rem(25);
+  padding-bottom: calc(3.125 * var(--space-unit));
```

Comment:
All good here but might be worth getting Jason's thoughts on the result item spacings. If we can introduce the new spacing standard quickly and easily then it'll bring things into step with his latest work.

Reply:
Agreed - will do